wiki/Beatmap/Difficulty_spread/en.md

```diff
@@ -0,0 +1,15 @@
+---
+tags:
+  - difficulty set
+  - progression
+---
+
+# Difficulty spread
+
+A **difficulty spread** is a set of [difficulties](/wiki/Beatmap/Difficulty) sorted by [star rating](/wiki/Beatmap/Star_rating) in ascending order, such that:
+
+- There are no missing [difficulty levels](/wiki/Beatmap/Difficulty#difficulty-levels) — every required difficulty between the easiest and the hardest ones from the spread's range is present.
+  - Difficulties lower than the lowest difficulty level required, as determined by [spread rules](/wiki/Ranking_Criteria#rules.1), can be considered as optional and excluded from spread requirements.
+- There are no significant difficulty gaps between two adjacent difficulties.
+
+The difficulty spread is a [ranking criteria](/wiki/Ranking_Criteria) requirement.
```

Comment:
the spread rules aren't requirements for something to be called a "difficulty spread"

```suggestion
A **difficulty spread** is a set of [difficulties](/wiki/Beatmap/Difficulty) in a [beatmap](/wiki/Beatmap) within some range of [star rating](/wiki/Beatmap/Star_rating) values.

[Ranked](/wiki/Beatmap/Category#ranked) beatmaps have several requirements regarding difficulty spread as per the [ranking criteria](/wiki/Ranking_Criteria):

- All required [difficulty levels](/wiki/Beatmap/Difficulty#difficulty-levels) within the spread's range must be present.
- Difficulties lower than the lowest difficulty level required, as determined by the [spread rules](/wiki/Ranking_Criteria#rules.1), are optional.
- No two adjacent beatmap difficulties may have a significant gap in difficulty.
```

Reply:
don't know if summarising parts of the ranking criteria in this article is a good idea though

Reply:
well, that is essentially what a spread is, so it's fine by me